doc/src/fix_colvars.txt

```diff
@@ -10,7 +10,7 @@ fix colvars command :h3
 
 [Syntax:]
 
-fix ID group-ID colvars configfile keyword values ... :pre
+fix ID group-ID Colvars configfile keyword values ... :pre
```

Comment:
No, it's carelessly semi-intentional: I got mixed up with the ID field, which I set to "Colvars" (upper case) in the library README.